src/docker.test.ts

```diff
@@ -69,6 +69,14 @@ describe("Docker images", (): void => {
     expect(core.getState).lastCalledWith(docker.CACHE_HIT);
   };
 
+  const assertsCacheHitSave = (): void => {
```

Comment:
Nit: usually verbs or verb phrases are preferred for function names, which in this case can be achieved by removing the first "s."